addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/MilightV6SessionManager.java

```diff
@@ -270,7 +270,7 @@ private void send_search_for_broadcast() {
                         Thread.sleep(10);
                         sendQueue.datagramSocket.send(p);
                     } catch (IOException e) {
-                        logger.error("Could not send discover packet! " + e.getLocalizedMessage());
+                        logger.error("Could not send discover packet! {}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace